### PR TITLE
add configs to dependabot yml

### LINKED ISSUE #2222: Configure dependabot yaml
The dependabot yaml is only a test file. Please add all necessary configs.

Find all details here https://docs.github.com/github/administering-a-repository/keeping-your-dependencies-updated-automatically

### PR DESCRIPTION
**Pull request by MohitGupta121**
Fixes #2222

### CODE REVIEW

**Comment by MohitGupta121** [comment]
Hii,
We have to need all the dependency updated or ignore some of them, so that they are compatible with our project???

I have make a pull request for that issue can you please review it.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,9 +4,19 @@
...
-updates:
-  - package-ecosystem: "" # See documentation for possible values
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "daily"
+# Updates for Github Actions used in the repo
+    - package-ecosystem: "github-actions"
+      directory: "/"
+      schedule:
+        interval: "daily"
```
Comment (CloudyPadmal) [inline .github/dependabot.yml:11]: Let’s change this to `weekly`

File: .github/dependabot.yml [hunk 2]
```diff
@@ -4,9 +4,19 @@
...
+      reviewers:
+      - "mariobehling"  
+# Updates for Gradle dependencies used in the app      
+    - package-ecosystem: gradle
+      directory: "/"
+      schedule:
+        interval: "daily"
```
Comment (CloudyPadmal) [inline .github/dependabot.yml:18]: Let’s change this to `weekly`

File: .github/dependabot.yml [hunk 3]
```diff
@@ -4,9 +4,19 @@
...
-      interval: "daily"
+# Updates for Github Actions used in the repo
+    - package-ecosystem: "github-actions"
+      directory: "/"
+      schedule:
+        interval: "daily"
+      reviewers:
+      - "mariobehling"  
+# Updates for Gradle dependencies used in the app      
+    - package-ecosystem: gradle
```
Comment (CloudyPadmal) [inline .github/dependabot.yml:15]:
```suggestion
    - package-ecosystem: “gradle“
```

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (MohitGupta121) [reply .github/dependabot.yml:11]: ok

File: .github/dependabot.yml (hunk 3 above)
Comment (MohitGupta121) [reply .github/dependabot.yml:15]: okay, Thanks

**Comment by MohitGupta121** [comment]
@CloudyPadmal can you now please check my new commit so that @mariobehling  can merge this.

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**